web/impact/impact/permissions.py

```diff
@@ -38,11 +47,6 @@ class DynamicModelPermissions(BasePermission):
     authenticated_users_only = True
 
     def has_permission(self, request, view):
-        # Workaround to ensure DjangoModelPermissions are not applied
-        # to the root view when using DefaultRouter.
-        if getattr(view, '_ignore_model_permissions', False):
-            return True
-
```

Comment:
It was not covered and as far as I can tell is never true.  I believe it is a remnant from the back when the impact-api was first getting developed.